app/views/logged_in.erb

```diff
@@ -0,0 +1,10 @@
+<span id='logged_in'>
+  <form action='/sessions/<%= session[:user] %>' method='post'>
+    <b>You're Logged In As: <%= session[:name] %>!</b>
```

Comment:
`<b>` is almost always presentational. Use CSS to style it with a different `font-weight`.